bokeh/core/_templates/autoload_nb_js.js

```diff
@@ -1,5 +1,102 @@
 {% extends "autoload_js.js" %}
 
+{% block register_mimetype %}
+
+  var MIME_TYPE = 'application/vnd.bokehjs_exec.v0+json';
+  var CLASS_NAME = 'output_bokeh rendered_html';
+
+  /**
+   * Render data to the DOM node
+   */
+  function render(props, node) {
+    var div = document.createElement("div")
+    div.innerHTML = props["data"]["div"]
+    var script = document.createElement("script")
+    script.textContent = props["data"]["script"]
+
+    node.appendChild(div);
+    node.appendChild(script);
+  }
+
+  /**
+   * Handle when an output is cleared or removed
+   */
+  function handleClearOutput(event, { cell: { output_area } }) {
```

Comment:
This syntax (destructuring) is not supported in IE. There are more instances to fix.